website/content/commands/namespace/create.mdx

```diff
@@ -31,10 +31,12 @@ from the CLI arguments.
   in this namespace. May be specified multiple times.
 
 - `-default-policy-name=<value>` - Name of a policy from the default namespace to inject for all
-  tokens in this namespace. May be specified multiple times.
+  tokens in this namespace. May be specified multiple times. Note that the ACL token used with
```

Comment:
Same comment about trimming 'Note that' and linking to the rule.
In the future we should write these using complete sentences instead of the stunted tech-caveman. 
We should also put this information in a table.